fdbclient/ClientKnobs.cpp

```diff
@@ -298,6 +298,7 @@ void ClientKnobs::initialize(Randomize randomize) {
 	init( CLIENT_ENABLE_USING_CLUSTER_ID_KEY,     false );
 
 	init( ENABLE_ENCRYPTION_CPU_TIME_LOGGING,     false );
+	init( EKP_TENANT_ID_TO_DROP,     TenantInfo::INVALID_TENANT );
```

Comment:
can we name it explicitly to make sure variable is only used in SimKms testing or testing in general?

Reply:
Also would be good to document the purpose of the KNOB to add more clarity

Reply:
I'll document it in the header file